.devcontainer/devcontainer.json

```diff
@@ -38,6 +47,50 @@
       "!include_dir_list scalar",
       "!include_dir_merge_list scalar",
       "!include_dir_merge_named scalar"
+    ],
+    // python common
+    "[python]": {
+      "editor.defaultFormatter": "ms-python.python",
+      "editor.codeActionsOnSave": {
+        "source.organizeImports": true
+      }
+    },
+    "python.linting.enabled": true,
+    "python.pythonPath": "/usr/local/bin/python",
+    // python linting
+    "python.linting.banditEnabled": true,
+    "python.linting.banditArgs": [
+      "--format=custom",
+      "--configfile=tests/bandit.yaml"
+    ],
+    "python.linting.flake8Enabled": true,
+    "python.linting.flake8Args": ["--config=./setup.cfg"],
+    "python.linting.mypyEnabled": true,
+    "python.linting.mypyArgs": [
+      "--follow-imports=silent",
+      "--ignore-missing-imports",
+      "--show-column-numbers",
+      "--no-pretty",
+      "--config-file=./mypy.ini"
+    ],
+    "python.linting.pylintEnabled": false,
```

Comment:
Why is pylint disabled?

Reply:
The standalone pylint extension (also still made by Microsoft) is preferred now. It is better. The pylint piece in the python integration sucks in comparison.

Reply:
It's in preview, lets not just yet.